cpp/benchmarks/string/convert_durations.cpp

```diff
@@ -14,92 +14,60 @@
  * limitations under the License.
  */
 
+#include <benchmarks/common/generate_input.hpp>
 #include <benchmarks/fixture/benchmark_fixture.hpp>
-#include <benchmarks/synchronization/synchronization.hpp>
-
-#include <cudf_test/column_wrapper.hpp>
 
 #include <cudf/column/column_view.hpp>
 #include <cudf/strings/convert/convert_durations.hpp>
-#include <cudf/types.hpp>
+#include <cudf/strings/strings_column_view.hpp>
 #include <cudf/wrappers/durations.hpp>
 
-#include <algorithm>
-#include <random>
-
-class DurationsToString : public cudf::benchmark {};
-template <class TypeParam>
-void BM_convert_from_durations(benchmark::State& state)
-{
-  cudf::size_type const source_size = state.range(0);
-
-  // Every element is valid
-  auto data = cudf::detail::make_counting_transform_iterator(
-    0, [source_size](auto i) { return TypeParam{i - source_size / 2}; });
+#include <nvbench/nvbench.cuh>
 
-  cudf::test::fixed_width_column_wrapper<TypeParam> source_durations(data, data + source_size);
+NVBENCH_DECLARE_TYPE_STRINGS(cudf::duration_D, "cudf::duration_D", "cudf::duration_D");
+NVBENCH_DECLARE_TYPE_STRINGS(cudf::duration_s, "cudf::duration_s", "cudf::duration_s");
+NVBENCH_DECLARE_TYPE_STRINGS(cudf::duration_ms, "cudf::duration_ms", "cudf::duration_ms");
+NVBENCH_DECLARE_TYPE_STRINGS(cudf::duration_us, "cudf::duration_us", "cudf::duration_us");
+NVBENCH_DECLARE_TYPE_STRINGS(cudf::duration_ns, "cudf::duration_ns", "cudf::duration_ns");
```

Comment:
Could we move this to a common header?

Reply:
I've since seen other usages they I feel would require a separate PR to resolve.
Example: https://github.com/rapidsai/cudf/pull/17194/files#diff-2fa34b25a2e1b46e1e15f633b539e9b930a7728b9159e5ec810dd9d0e17bc50aR159-R219